renderdocui/Windows/MainWindow.cs

```diff
@@ -1006,18 +1006,32 @@ private LiveCapture OnCaptureTrigger(string exe, string workingDir, string cmdLi
             if (!PromptCloseLog())
                 return null;
 
-            string logfile = m_Core.TempLogFilename(Path.GetFileNameWithoutExtension(exe));
-
-            UInt32 ret = m_Core.Renderer.ExecuteAndInject(exe, workingDir, cmdLine, env, logfile, opts);
+            UInt32 remoteIdent;
+            if (m_Core.Renderer.Remote != null && m_Core.Renderer.Remote.Hostname.StartsWith("adb:"))
```

Comment:
Ok I think I can execute those shell commands from android native code in ExecuteAndInject.